include/rocksdb/statistics.h

```diff
@@ -576,6 +576,9 @@ class Statistics : public Customizable {
   static Status CreateFromString(const ConfigOptions& opts,
                                  const std::string& value,
                                  std::shared_ptr<Statistics>* result);
+  // Default name of empty, for backwards compatibility.  Derived classes should
+  // override this method.
```

Comment:
Suggest adding "This default implementation will likely be removed in a future release" so that we can grep for stuff like "future release" when we're ready for a round of breaking API changes.